deployment/deploy_ingestion.sh

```diff
@@ -128,8 +137,8 @@ aws iam attach-role-policy --policy-arn ${S3_READ_WRITE_POLICY} --role-name lamb
 wait
 
 echo 'Creating lambda function...'
-FUNCTION=$(aws lambda create-function --function-name ${FUNCTION_NAME} --runtime python3.9 --role ${IAM_ROLE} --package-type Zip --handler test-ingestion.lambda_handler --code S3Bucket=${CODE_BUCKET_NAME},S3Key=${FUNCTION_NAME}/test-ingestion.zip)
-sleep 10
+FUNCTION=$(aws lambda create-function --function-name ${FUNCTION_NAME} --runtime python3.9 --role ${IAM_ROLE} --package-type Zip --handler function.lambda_handler --code S3Bucket=${CODE_BUCKET_NAME},S3Key=${FUNCTION_NAME}/test-ingestion.zip)
+wait
```

Comment:
Lambda handler name change might cause issues

Reply:
Renamed test-ingestion.py to function.py to fix this